soh/soh/Enhancements/randomizer/settings.cpp

```diff
@@ -787,6 +788,7 @@ void Settings::CreateOptions() {
         &mOptions[RSK_FISHSANITY_POND_COUNT],
         &mOptions[RSK_FISHSANITY_AGE_SPLIT],
         &mOptions[RSK_SHUFFLE_SCRUBS],
+        &mOptions[RSK_SHUFFLE_POTS],
```

Comment:
Merging caused the pot shuffle option to get wedged between the scrubsanity and scrubs price options, which is obviously bad.